### ISSUE TITLE
Add color coding to reflect recalculated Clinical Significance

### ISSUE THREAD

**Issue by toneillbroad** [clingen-data-model/clinvar-ingest-reports#43]
- [ ] Add color-coding to reflect the recalculated overall clinical significance (ask Danielle or Marina about this - need to add new column to BQ extract - need more info)

**Comment by dazzariti** [comment]
Update from Danielle to provide more context on the request and priority

This request was collected back in 2022. It will need further specification from the ClinGen Curation of ClinVar Project team and is not yet ready to prioritize.

"Recalculated overall clinical significance" refers to a clin sig value Larry provides once "Non-contributory" SCV flags from the ClinVar Curation Project have been removed the the overall clinical significance of a VCV. 

We haven't yet finalized how we would like VCEPs to prioritize this information so I think we can put this on hold until we do. I have added myself as an assignee to lead specification.
